projects/demo/src/modules/components/input-date-range/examples/4/index.html

```diff
@@ -0,0 +1,11 @@
+<h3>Control's output as native Date</h3>
+
+<tui-input-date-range class="b-form" [formControl]="control">
+    Choose dates
+</tui-input-date-range>
+
+<p>
+    <strong>Stringified control value:</strong>
+    <br />
```

Comment:
nit: we usually avoid br. or is this a valid case for it?

Reply:
Problem with `br` in my opinion is that it blends separation of concern between HTML and CSS. I think we should not use it at all, because our examples promote particular coding style and we should strive for them to be exemplary.

Reply:
I removed all `br`-s